src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -1202,10 +1202,39 @@ func getTempUrls(securityScopedUrls: [URL]) -> [URL] {
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
+### Specify a bucket
+
+We also have the ability to target a specific bucket by passing the bucket option to the `uploadData` request. The bucket option can either be A string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console.. 
```

Comment:
```suggestion
You can also perform upload operation against a specific bucket by providing `bucket` option. It can either be a string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console.
```